### PR TITLE
Update ccv0 demo instructions

### PR DESCRIPTION
**Pull request by bpradipt**
The series is to ensure a workable setup when using`ccv0-demo` branch.
It merges key commits from `main` to `ccv0-demo` as well as points to correct images 
- Operator: quay.io/confidential-containers/operator:demo
- Payload: quay.io/confidential-containers/runtime-payload:v0

Tested the instructions successfully on a 2 node K8s cluster running on Ubuntu 20.04
```
kubectl get nodes -o wide

NAME               STATUS   ROLES                  AGE   VERSION   INTERNAL-IP      EXTERNAL-IP   OS-IMAGE             KERNEL-VERSION     CONTAINER-RUNTIME
testk8s-master-0   Ready    control-plane,master   18h   v1.24.0   192.168.121.37   <none>        Ubuntu 20.04.3 LTS   5.4.0-91-generic   containerd://1.6.4
testk8s-worker-0   Ready    worker                 18h   v1.24.0   192.168.121.17   <none>        Ubuntu 20.04.3 LTS   5.4.0-91-generic   containerd://1.6.0-beta.0-512-gc8f5e4509
```

```
kubectl get pods -n confidential-containers-system

NAME                                              READY   STATUS    RESTARTS   AGE
cc-operator-controller-manager-5f45c6878d-8qbdq   2/2     Running   0          49m
cc-operator-daemon-install-gq5wv                  1/1     Running   0          49m
[kcli@virtlab714 operator]$ kubectl get pods -n confidential-containers-system -o wide
NAME                                              READY   STATUS    RESTARTS   AGE   IP            NODE               NOMINATED NODE   READINESS GATES
cc-operator-controller-manager-5f45c6878d-8qbdq   2/2     Running   0          49m   10.244.1.16   testk8s-worker-0   <none>           <none>
cc-operator-daemon-install-gq5wv                  1/1     Running   0          49m   10.244.1.17   testk8s-worker-0   <none>           <none>
```

```
kubectl get runtimeclass

NAME        HANDLER     AGE
kata        kata        49m
kata-cc     kata-cc     49m
kata-qemu   kata-qemu   49m
```

```
kubectl get pods
NAME                                   READY   STATUS    RESTARTS   AGE
nginx-deployment-cc-5c4d54569f-4wlqg   1/1     Running   0          47m

kubectl get pods -o yaml | grep -i runtimeclass
runtimeClassName: kata-cc

kubectl exec -it nginx-deployment-cc-5c4d54569f-4wlqg -- bash
error: Internal error occurred: error executing command in container: failed to exec in container: failed to start exec "b0f8453fd791bbaa5de1f3decfa50c3a45475be1cedcc68ca560debdacee9099": cannot enter container 830f6da3ed785ed7f082c11482610e307f9b6aa5f18fe6c1be36c165b578f31e, with err rpc error: code = Unimplemented desc = ExecProcessRequest is blocked: unknown
```

### CODE REVIEW

**Comment by bpradipt** [comment]
@surajssd this PR should fix the issues you had when trying out the instructions.

**Review by fidencio — APPROVED** [review]
lgtm, thanks @bpradipt!

**Comment by fidencio** [comment]
@bpradipt, some of the commits are missing your `Signed-off-by`, mind to do add that?

**Comment by bpradipt** [comment]
> @bpradipt, some of the commits are missing your `Signed-off-by`, mind to do add that?


> @bpradipt, some of the commits are missing your `Signed-off-by`, mind to do add that?

Done. Thanks @fidencio

File: docs/INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+## Installation
+
+Ensure KUBECONFIG points to the target Kubernetes cluster
+```
+kubectl apply -f https://raw.githubusercontent.com/confidential-containers/operator/ccv0-demo/deploy/deploy.yaml
+```
+
+## Create Custom Resource (CR)
+```
+kubectl apply  -f https://raw.githubusercontent.com/confidential-containers/operator/ccv0-demo/config/samples/ccruntime.yaml
```
Comment (surajssd) [inline docs/INSTALL.md:10]: This PR uses a [file](https://raw.githubusercontent.com/confidential-containers/operator/ccv0-demo/config/samples/ccruntime.yaml) from `ccv0-demo` branch. But the [current demo doc](https://github.com/confidential-containers/operator/blob/ccv0-demo/demo/README.md#install-confidential-containers-runtime) on the `ccv0-demo` points to the [config](https://raw.githubusercontent.com/confidential-containers/operator/main/config/samples/ccruntime.yaml) from `main` branch.

Is this intentional?

File: docs/INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+```
+
+## Verify the RuntimeClasses
+```
+kubectl get runtimeclass
```
Comment (surajssd) [inline docs/INSTALL.md:15]: I had to label the node to get things moving forward:

```
kubectl label node <node-name> node-role.kubernetes.io/worker=
```

**Comment by surajssd** [comment]
This document works for me. Shouldn't this PR remove the file: https://github.com/confidential-containers/operator/blob/ccv0-demo/demo/README.md ?

**Comment by bpradipt** [comment]
> This document works for me. Shouldn't this PR remove the file: https://github.com/confidential-containers/operator/blob/ccv0-demo/demo/README.md ?

Thanks for trying out. Yes, I plan to remove the `demo` folder completely. Will do it in a follow-on  PR

File: docs/INSTALL.md (hunk 1 above)
Comment (bpradipt) [reply docs/INSTALL.md:10]: This is intentional. I'll remove the `demo` folder in a follow on PR.

**Pull request merged**